dags/vm_resource.py

```diff
@@ -133,3 +137,5 @@ class DockerImage(enum.Enum):
   XPK_MAXTEXT_TEST = "gcr.io/tpu-prod-env-multipod/xpk_maxtext_test:latest"
   MAXTEXT_JAX_STABLE = f"gcr.io/tpu-prod-env-multipod/maxtext_jax_stable:{datetime.datetime.today().strftime('%Y-%m-%d')}"
```

Comment:
Nit: should we update this to `MAXTEXT_JAX_TPU_STABLE` then? same for `MAXTEXT_JAX_NIGHTLY`